### PR TITLE
Inputs not outputs

### LINKED ISSUE #200: Validate inputs != outputs names
https://github.com/CloudSlang/cloud-slang/issues/653

### PR DESCRIPTION
**Pull request by SamMarkowitz**
Fixes #200 

As per CloudSlang/cloud-slang#684

### CODE REVIEW

**Comment by SamMarkowitz** [comment]
Waiting for CloudSlang/cloud-slang#684 to have a CLI to test the code against.

**Comment by SamMarkowitz** [comment]
@gituser173 please review but ignore the build failure. Will be fixed with the next PR relating to parallel_loop.

File: cloudslang_dsl_reference.rst [hunk 1]
```diff
@@ -853,7 +852,9 @@ input name may in turn be mapped to its properties or an input
 `expression <#expressions>`__.
 
 Inputs are used to pass parameters to `flows <#flow>`__ or
-`operations <#operation>`__.
+`operations <#operation>`__. Input names for a specific `flow <#flow>`__ or
+`operation <#operation>`__ must be different than the `output <#outputs>`__
+names of the same `flow <#flow>`__ or `operation <#operation>`__.
```
Comment (gituser173) [inline cloudslang_dsl_reference.rst:859]: would also write about the fact that step inputs should be named different than the outputs of the flow/operation used in that step

Comment (SamMarkowitz) [inline cloudslang_dsl_reference.rst:859]: Are the step argument names not the same as the calling flow/operation inputs? So if a flow/operation's inputs and output  names are different, then the step argument names will be different as well.

**Comment by SamMarkowitz** [comment]
@gituser173 I see this one was back on me. Did you see my comment? Do you still think I need to mention something about the step inputs?

File: cloudslang_dsl_reference.rst (hunk 1 above)
Comment (SamMarkowitz) [inline cloudslang_dsl_reference.rst:859]: added in `do`

**Comment by gituser173** [comment]
:+1:

**Pull request merged**